normalize-own-data/README.md

```diff
@@ -2,6 +2,35 @@
 
 This directory contains an example workflow of how to quantile normalize your own dataset to make it more comparable to data you obtain from [refine.bio](https://www.refine.bio).
 
+## Requirements and usage
+
+This module requires you to install the following software to run the example yourself:
+
+* [**R**](https://cran.r-project.org/)
+* [**Bioconductor**](https://bioconductor.org/install/)
+* [**tidyverse**](https://www.tidyverse.org/)
+* [**RStudio**](https://www.rstudio.com/products/RStudio/) for working with [R Notebooks](https://bookdown.org/yihui/rmarkdown/notebook.html).
+* [**devtools**](https://cran.r-project.org/web/packages/devtools/readme/README.html) will be required for installing some packages from GitHub. We recommend installing `devtools` from CRAN.
+* **wget** - [installation instructions for Mac with Homebrew](https://www.maketecheasier.com/install-wget-mac/), [installation instructions for Windows 10](https://builtvisible.com/download-your-website-with-wget/) (under _Install WGET in Windows 10_), [installation instructions for Debian or Ubuntu Linux](https://www.cyberciti.biz/faq/how-to-install-wget-togetrid-of-error-bash-wget-command-not-found/)
```

Comment:
Do we `wget` anywhere else that we might need to add these instructions?

Reply:
Not that I am aware of